### DISCUSSION TITLE
Work on WSL2 Ubuntu

### DISCUSSION THREAD

**Discussion by navalex** [api-platform/api-platform#2076] [Q&A]
Hello,

I'm trying to setup my first API Platform project on my Windows 11 computer. All my project are running on my WSL2 Ubuntu installation, and some of them in a docker container. So I didn't expected any problem by install API platform on the same install.

I use the API Platform Distribution, all containers launch well, and I can sometime load https://localhost, but only one time after docker build, the other try load infinitely. So I tried to make another install outside WSL and it seem to work without problem (except SSL certificate trust but anyway I don't need to trust it for now).

So wanted to know if someone have an installation like mine and manage to make it work ?

(should I try with another distribution ?)

**Comment by valentin74** [comment]
Hello,
Just to be sure, are you running your project under \\wsl$\ path?

**Comment by lxix** [comment]
Hello,
Seems like there is a problem with [caddy and go](https://github.com/caddyserver/caddy/issues/3896). For me, switching to firefox "solved" the problem.
